pkg/frontend/querymiddleware/roundtrip_test.go

```diff
@@ -186,26 +186,6 @@ func TestTripperware_InstantQuery(t *testing.T) {
 		}, res)
 	})
 
-	t.Run("specific time param with form being already parsed", func(t *testing.T) {
```

Comment:
these tests are for [old code](https://github.com/grafana/mimir/pull/3599) that was [removed](https://github.com/grafana/mimir/pull/7026) as it is more robustly handled in the prometheus codec and associated utils.

The tests continued to pass by chance that we weren't using this particular request parsing middleware twice.

We have moved away from trying to go back and fix the situation if the request body is consumed by our own middleware. Instead everything that reads a request body just needs to use `ParseRequestFormWithoutConsumingBody` introduced in https://github.com/grafana/mimir/pull/5524.